bigquery_etl/docs/bqetl/generate_bqetl_docs.py

```diff
@@ -0,0 +1,96 @@
+"""Generate docs for bqetl commands."""
+
+import click
+from jinja2 import Environment, FileSystemLoader
+import os
+from pathlib import Path
+
+from bigquery_etl.cli.dag import dag
+from bigquery_etl.cli.dryrun import dryrun
+from bigquery_etl.cli.format import format
+from bigquery_etl.cli.query import query
+from bigquery_etl.cli.routine import mozfun, routine
+from bigquery_etl.dependency import dependency
+
+
+# commands to document
+COMMANDS = {
+    "query": query,
+    "dag": dag,
+    "dependency": dependency,
+    "dryrun": dryrun,
+    "format": format,
+    "routine": routine,
+    "mozfun": mozfun,
+}
+FILE_PATH = Path(os.path.dirname(__file__))
+BASE_DIR = FILE_PATH.parent.parent
+TEMPLATE = FILE_PATH / "templates" / "commands.md"
+
+
+def extract_description_from_help_text(help_text):
+    """Return the description from the command help text."""
+    return help_text.split("\n\n")[0].strip()
+
+
+def extract_examples_from_help_text(help_text):
+    """Return the examples from the command help text."""
+    examples = None
+    help_text = help_text.split("\n\n")
+
+    if len(help_text) > 1:
+        examples = "\n\n".join(help_text[1:])
+        examples = examples.replace("Examples:\n\n", "")
+        examples = examples.replace("\b", "")
```

Comment:
Removing `\b` here. It shows up in the rendered Markdown as newlines

Reply:
Can you add this concern as a code comment and link to https://click.palletsprojects.com/en/7.x/documentation/#preventing-rewrapping